ingredient_phrase_tagger/training/tokenizer.py

```diff
@@ -1,16 +1,25 @@
+#!/usr/bin/env python3
```

Comment:
We don't need this, as the shebang line is only for executable files.

This project follows Google Python conventions:

https://github.com/google/styleguide/blob/gh-pages/pyguide.md#37-shebang-line